prettier.go

```diff
@@ -0,0 +1,170 @@
+package metricsql
+
+import (
+	"fmt"
+	"strings"
+)
+
+const indentString = "  "
+
+func paddings(indent int) string {
+	var str strings.Builder
+	for i := 0; i < indent; i++ {
+		str.WriteString(indentString)
+	}
+	return str.String()
+}
+
+func shouldWrap(expr Expr) bool {
+	switch expr.(type) {
+	case *MetricExpr, *NumberExpr, *StringExpr, *FuncExpr:
+		return false
+	default:
+		return true
+	}
+}
+
+func wrapWithBraces(expr Expr, b *strings.Builder, indent, maxLineLength int) {
+	paddings := paddings(indent)
```

Comment:
```suggestion
	padding := paddings(indent)
```
nit: Plural form seems to be misleading here

Reply:
updated behavior of the function